### DISCUSSION TITLE
Adding Background Image In A Website Page

### DISCUSSION THREAD

**Discussion by samarzayan** [quarto-dev/quarto-cli#2962] [Q&A] [answered]
As an example, the Quarto [home](https://quarto.org/) page has a background image with text. I am trying to do something similar on my home page. I tried reading the documentation, but I got confused. Can someone please help?

Thanks in advance!

**Comment by fredguth** [comment, marked as answer]
You can check the code for Quarto's website here: [quarto-web](https://github.com/quarto-dev/quarto-web)

Notice that you can create a `div` using the syntax
```markdown
::: {.class-name}

content here

:::
```
Also, you can change the body background in styles.css.

I created a minimum working example in this repo: https://github.com/fredguth/demo-quarto-background

**Reply by samarzayan** [reply]
Thanks so much! This solved my problem.
